### DISCUSSION TITLE
Welcome to quiz-app Discussions!

### DISCUSSION THREAD

**Discussion by struckchure** [Dev-47/quiz-app#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by struckchure** [comment]
And you might want to notify the necessary teams or person involved in a case where you're making heavy changes

Drop 'em here

**Reply by dchief1** [reply]
Hello, I'm Kelechi 
I'm part of the Frontend team 
I want to know my team members , thank you ☺️

**Reply by struckchure** [reply]
Hey, I'm Mohammed
Part of the FE and BE team, good to have you here chief

**Comment by horlakz** [comment]
niceee
